docs/getting_started_linux_cmake.md

```diff
@@ -0,0 +1,114 @@
+# Getting Started on Linux with CMake
+
+<!--
+Notes to those updating this guide:
+
+    * This document should be __simple__ and cover essential items only.
+      Notes for optional components should go in separate files.
+
+    * This document parallels getting_started_windows_cmake.md.
+      Please keep them in sync.
+-->
+
+## Prerequisites
+
+### Install CMake
+
+Install CMake version >= 3.13:
+
+```shell
+$ sudo apt install cmake
+```
+
+> Tip:<br>
+> &nbsp;&nbsp;&nbsp;&nbsp;Your editor of choice likely has plugins for CMake,
+such as the Visual Studio Code
+[CMake Tools](https://github.com/microsoft/vscode-cmake-tools) extension.
+
+### Install a Compiler
+
+We recommend Clang. GCC is not fully supported.
+
+```shell
+$ sudo apt install clang
+```
+
+Verify the version with `clang++ --version`. We have tested with the following
+versions:
+
+*   6.0.1
+*   8.0.1
+
+There have been issues with the following versions:
+
+*   [9.0.0-2](https://github.com/google/iree/issues/1216)
```

Comment:
The issue mentions **9.0.1**, not **9.0.0-2**. Anyway, neither had a problem on my Gentoo machine with 9.0.1 so far nor with the Ubuntu VM using 9.0.0-2...

Reply:
I'll just remove the clang compatibility section then. It doesn't seem as useful if there's only one report of a version not working.